app/models/place.rb

```diff
@@ -29,4 +33,52 @@ def self.prefectures_with_key
       }
     end
   end
+
+  def self.create_index!(options={})
+    client = __elasticsearch__.client
+    settings index: {
+               analysis: {
+                 tokenizer: {
+                   ngram_tokenizer: {
+                     type: 'nGram',
+                     min_gram: '2',
+                     max_gram: '3',
+                     token_chars: ['letter', 'digit']
+                   }
+                 },                 
+                 analyzer: {
+                   ngram_analyzer: {
+                     tokenizer: "ngram_tokenizer"
+                   },
+                   kuromoji_analyzer: {
+                     tokenizer: 'kuromoji_tokenizer',
+                     filter:    ['kuromoji_baseform'],
+                   }
+                 }
+               }
+             } do
+      mapping _source: { enabled: true },
+      properties: { enabled: true, analyzer: 'kuromoji_analyzer' } do
+        indexes :trendword_id, type: :integer, index: 'not_analyzed'
+        indexes :name, type: :string, index: 'analyzed', analyzer: 'kuromoji_analyzer'
+        indexes :location, type: :geo_point, lat_lon: true
+      end
+    end
+    client.indices.delete index: 'places' rescue nil if options[:force]
+    client.indices.create index: 'places',
+      body: {
+        settings: settings.to_hash,
+        mappings: mappings.to_hash
+      }
+  end
+  def as_indexed_json(options = {})
```

Comment:
__[rubocop]__
* Use empty lines between method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods)
* Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `as_indexed_json(*)` if you want the method to accept any arguments but don't care about them. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)